lib/metadata/virtual_container_host.go

```diff
@@ -63,14 +63,15 @@ type VirtualContainerHostConfigSpec struct {
 	HostCertificates map[string]tls.Certificate
 
 	// Port Layer - storage
-	// Datastore URLs for image stores - the top layer is [0], the bottom layer is [len-1]
-	ImageStores []string `vic:"0.1" scope:"read-only" key:"image_stores"`
+	// Datastore used for the ImageStore
+	// TODO this will change, probably to url.URL, as needed by the port layer
+	ImageStore string `vic:"0.1" scope:"read-only" key:"image_store"`
 	// Permitted datastore URL roots for volumes
-	VolumeLocations []url.URL `vic:"0.1" scope:"read-only" recurse:"depth=0"`
+	VolumeLocations []types.ManagedObjectReference `vic:"0.1" scope:"read-only" recurse:"depth=0"`
```

Comment:
So the image datastore attributes are still plain string.
This is fine, but as we'll move all datastore related attributes to URL eventually, you might not change these definitions to ManagedObjectReference.